functionalTest/wdio.conf.js

```diff
@@ -0,0 +1,298 @@
+exports.config = {
+    //
+    // ====================
```

Comment:
Once we get comfortable with WDIO, I'd suggest removing all this description of the parameters in this file, and if necessary just replacing it with a link to the appropriate documentation. Keeps things tidy.

Reply:
Removed some description from the config file which is not required. Only keeping the descriptions which might be helpful for reference.